src/views/Vote/CreateProposal/CreateProposal.tsx

```diff
@@ -0,0 +1,172 @@
+/** @jsxImportSource theme-ui */
+import React, { useState } from 'react'
+import { Box, Flex, Text } from 'theme-ui'
+import { Svg, ArrowBackIcon, Link, Textarea, Button, Spinner, TooltipBubble, Input } from '@ape.swap/uikit'
+import { create } from 'ipfs-http-client'
+import { MarkdownIcon } from 'components/Icons'
+import { styles } from './styles'
+import Preview from './Preview'
+
+export interface InputProps {
+  onChange: (e: React.FormEvent<HTMLInputElement>) => void
+  placeholder?: string
+  value: string
+}
+
+const CreateProposal = () => {
+  const [proposalData, setPropposalData] = useState({
+    title: '',
+    dicussion: '',
+    textarea: '',
+  })
+
+  const [state, setState] = useState({
+    preview: true,
```

Comment:
split this state into two, they are unrelated

Reply:
Done